compose.d/tasks.yml

```diff
@@ -66,6 +66,6 @@ services:
           acf-to-rest-api=3.1.0
           wordpress-importer=0.6.4
           disable-comments=1.10.2
-          title-and-nofollow-for-links=1.09
-          wp-security-audit-log=4.0.0
-          activity-log-mainwp=1.4.1
+          title-and-nofollow-for-links=1.10
+          wp-security-audit-log=4.1.2
+          # activity-log-mainwp=1.4.1
```

Comment:
Does not seem to be related to current PR.

Reply:
OK. I understand. Better to remove pipefail entirely?

Reply:
It is not. Let was just experimenting, would remove.